lib/models/blueprint.js

```diff
@@ -1379,12 +1381,18 @@ Blueprint.renamedFiles = {
 
 /**
   @static
-  @property ignoredFiles
+  @property initialIgnoredFiles
 */
-Blueprint.ignoredFiles = [
+Blueprint.initialIgnoredFiles = [
```

Comment:
We can store this off the prototype at the top of the file. No need to expose this internal detail.

Reply:
Thanks @kellyselden, I updated it